StudioCore/ParamEditor/ParamBank.cs

```diff
@@ -1018,6 +1018,7 @@ public static void SaveParams(bool loose = false, bool partialParams = false)
             {
                 SaveParamsER(partialParams);
             }
+            TaskManager.UpdateLastActionString("Saved Parameters");
```

Comment:
Again we need a better system than just some spread out state and global variable. I'd hold off on this.